### ISSUE TITLE
google hangout

### ISSUE THREAD

**Issue by ai10** [GravityProject/gravity#50]
maybe we can schedule a hangout to discuss roadmap, architecture thoughts etc.?

I have some strong opinions about a 'social organization' application which is a little bit different from a 'social network' concept.

**Comment by pmuens** [comment]
That is a good idea. I have also some questions about the future of the project. Whether it should be a social network especially for Meteor guys, or a more general place to hang out where "ordinary" people would also sign up.

Maybe we should make some noise and gather other people as well to discuss the future of Gravity.

**Comment by pmuens** [comment]
@8mylez @michaltakac guys, what do you think?

**Comment by 8mylez** [comment]
Sounds good!
Philipp Muens <notifications@github.com> schrieb am Do., 12. Nov. 2015 um
21:09:

> @8mylez <https://github.com/8mylez> @michaltakac
> <https://github.com/michaltakac> guys, what do you think?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GravityProject/gravity/issues/50#issuecomment-156220341>
> .
>
-- 
Herzlichen Dank!

mit den besten Grüßen aus Paderborn

*Alexander Wolf*
Geschäftsführender Gesellschafter

*Telefon:* +49 5251 7776630

*E-Mail:* alexander.wolf@goltfisch.de / i@8mylez.com
*Web:* goltfisch.de / 8mylez.com

Goltfisch GmbH
Lesteweg 9
33106 Paderborn

Geschäftsführer: Marc Baur, Philipp Müns, Alexander Wolf
Amtsgericht Paderborn, HRB 11796
USt-IdNr. DE299503696, St.Nr. 339/5818/1693

**Comment by michaltakac** [comment]
Would be cool :+1:

**Comment by josephdburdick** [comment]
@pmuens Think big. This could be the open-source rival of http://Ello.co

**Comment by pmuens** [comment]
@josephdburdick yes, I think that this should have big ambitions and should be open for everyone (not just developers).

Let's build something huge!

**Comment by zaverichintan** [comment]
Hey!  Kindly have hangout every month or fortnight, so that new devs like me can join in. 
It will be more fun..
